keyboards/crkbd/keymaps/blipson/rules.mk

```diff
@@ -0,0 +1,4 @@
+OLED_ENABLE = yes
+OLED_DRIVER = SSD1306
```

Comment:
This should be specified at the keyboard level, so shouldn't need it here, I think.
```suggestion
```